scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -86,14 +128,29 @@ def main(args):
     """
     Handles the command line arguments.
 
+    Parameters
+    ----------
     args: The list of arguments passed.
-    return:
-        (int) value to return to the OS as program exit code.
+
+    Returns
+    -------
+    (int)
+        value to return to the OS as program exit code.
     """
     parser = argparse.ArgumentParser()
     parser.add_argument('--debug', action="store_true", default=False)
-    parser.add_argument("--role", dest="role", default="diagnostic-ccd", metavar="<component>",
-                        help="Role of the camera to connect to via the Odemis back-end. Ex: 'ccd'.")
+    parser.add_argument("--role", dest="role", default="diagnostic-ccd",
+                        metavar="<component>",
+                        help="Role of the camera to connect to via the Odemis "
+                             "back-end. Ex: 'ccd'.")
+    parser.add_argument("--channel", dest="channel", default=1,
+                        metavar="<component>",
+                        help="Number of the AWG output channel.")
+    parser.add_argument("--auto", dest="auto", default=True,
+                        metavar="<component>",
+                        help="If True automatically align the ebeam scan to"
+                             "multiprobe. To do this a server must be running "
+                             "on the microscope PC.")
 
```

Comment:
Would that need some more info here? Explaining which server? I am not sure I know what to do, but maybe one would assume the regular user would know.

Reply:
Where can I find this document?

Reply:
for now it's only on the desktop of the sonic control PC, but I was planning on adding it somewhere on git or dropbox today (just need to decide where to put it).